### PR TITLE
adding local batch example

### PR DESCRIPTION
**Pull request by edshee**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/source/contents/examples/batch-examples-local.md [hunk 1]
```diff
@@ -0,0 +1,648 @@
+# Local Batch Inference Example
+
+This example runs you through a series of batch inference requests made to both models and pipelines running on Seldon Core locally. 
+
+## Setup
```
Comment (cliveseldon) [inline docs/source/contents/examples/batch-examples-local.md:5]: I think we shouldn't repeat install docs but just link to existing place in docs

File: docs/source/contents/examples/batch-examples-local.md [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+  steps:
+    - name: tfsimple1
+  output:
+    steps:
+    - tfsimple1
+
+```
+and deploy it:
+```bash
+seldon pipeline load -f pipelines/tfsimple.yaml
```
Comment (cliveseldon) [inline docs/source/contents/examples/batch-examples-local.md:138]: You could use the seldon CLI to wait for model or pipeline ready as done in other examples?

**Review by cliveseldon — CHANGES_REQUESTED** [review]

File: docs/source/contents/examples/batch-examples-local.md (hunk 1 above)
Comment (edshee) [reply docs/source/contents/examples/batch-examples-local.md:5]: Good point, I'll just link to that section and remove this

File: docs/source/contents/examples/batch-examples-local.md (hunk 2 above)
Comment (edshee) [reply docs/source/contents/examples/batch-examples-local.md:138]: I quite wanted to show the `model list` and `pipeline list` functions of the CLI though, which I do underneath. That way people get a variety of different ways to see things across the examples.

**Review by cliveseldon — APPROVED** [review]

**Pull request merged**